src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -274,6 +404,34 @@ def build_process_costs(
             transport_cost_per_ton_product: Expression, Var or Param to use for transport costs
                 per ton of product (note, this is not part of the TOC)
             CE_index_year: year for cost basis, e.g. "2021" to use 2021 dollars
+            watertap_block: list of unit model blocks corresponding to watertap models
+            calculate_NPV: True/false flag for calculating net present value (NPV).
+
+            Keyword arguments related to NPV, entered at class instantiation:
```

Comment:
All of this should already be included in the main class doc string from the config block (and as mentioned above I think all these arguments are just duplicating effort).